posixlib/src/main/resources/scala-native/netdb.c

```diff
@@ -48,13 +48,13 @@ void scalanative_convert_addrinfo(struct addrinfo *in,
         socklen_t size;
         if (in->ai_addr->sa_family == AF_INET) {
             struct scalanative_sockaddr_in *addr =
-                malloc(sizeof(struct scalanative_sockaddr_in));
+                calloc(1, sizeof(struct scalanative_sockaddr_in));
```

Comment:
Why clear memory that is just going to be overwritten?  
Network code is particularly sensitive to speed of execution concerns and regressions.

I believe that all of the `calloc` in this file do not increase correctness and decrease 
performance and should not happen.